### PR TITLE
OSDOCS 7460: Modified the note in 'About cert manager section'

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.12+

Issue: [OSDOCS 7460](https://issues.redhat.com/browse/OSDOCS-7460)

Link to docs preview: [Preview](https://70743--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/#cert-manager-about_cert-manager-operator-about)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cert-manager-about.adoc [hunk 1]
```diff
@@ -17,5 +17,7 @@ The {cert-manager-operator} provides the following features:
 
 [IMPORTANT]
 ====
-Do not attempt to use more than one cert-manager Operator in your cluster. If you have a community cert-manager Operator installed in your cluster, you must uninstall it before installing the {cert-manager-operator}.
+Do not attempt to use both cert-manager Operator for {product-title} and community {cert-manager-operator} at the same time in your cluster.
```
Comment (lunarwhite) [inline modules/cert-manager-about.adoc:20]:
```suggestion
Do not attempt to use both {cert-manager-operator} for {product-title} and community cert-manager operator at the same time in your cluster.
```

File: modules/cert-manager-about.adoc [hunk 2]
```diff
@@ -17,5 +17,7 @@ The {cert-manager-operator} provides the following features:
...
+
+Also you not install multiple {cert-manager-operator} for {product-title} in different namespace in one OpenShift cluster.
```
Comment (lunarwhite) [inline modules/cert-manager-about.adoc:22]:
```suggestion
Also you should not install multiple {cert-manager-operator} for {product-title} in different namespaces within one OpenShift cluster.
```

**Review by lunarwhite — COMMENTED** [review]
Left minor wording suggestions. Otherwise LGTM.

File: modules/cert-manager-about.adoc (hunk 1 above)
Comment (lunarwhite) [reply modules/cert-manager-about.adoc:20]: I think it should be fine to not remove the second sentence: `If you have a community cert-manager Operator installed in your cluster, you must uninstall it before installing the {cert-manager-operator}.`

**Comment by lunarwhite** [comment]
Hi @subhtk, it would be great if we could merge this one for customers. Could you PTAL when you have time? Thanks!

Also [this one](https://github.com/openshift/openshift-docs/pull/70742) ^^

File: modules/cert-manager-about.adoc (hunk 2 above)
Comment (subhtk) [reply modules/cert-manager-about.adoc:22]: ack

File: modules/cert-manager-about.adoc (hunk 1 above)
Comment (subhtk) [reply modules/cert-manager-about.adoc:20]: ack.

File: modules/cert-manager-about.adoc [hunk 3]
```diff
@@ -17,5 +17,7 @@ The {cert-manager-operator} provides the following features:
 
 [IMPORTANT]
 ====
-Do not attempt to use more than one cert-manager Operator in your cluster. If you have a community cert-manager Operator installed in your cluster, you must uninstall it before installing the {cert-manager-operator}.
+Do not attempt to use both {cert-manager-operator} for {product-title} and community cert-manager operator at the same time in your cluster.
```
Comment (bhardesty) [inline modules/cert-manager-about.adoc:20]:
```suggestion
Do not attempt to use both {cert-manager-operator} for {product-title} and the community cert-manager Operator at the same time in your cluster.
```

File: modules/cert-manager-about.adoc [hunk 4]
```diff
@@ -17,5 +17,7 @@ The {cert-manager-operator} provides the following features:
...
+
+Also you should not install multiple {cert-manager-operator} for {product-title} in different namespaces within one OpenShift cluster.
```
Comment (bhardesty) [inline modules/cert-manager-about.adoc:22]:
```suggestion
Also, you should not install {cert-manager-operator} for {product-title} in multiple namespaces within a single OpenShift cluster.
```

**Review by bhardesty — COMMENTED** [review]
Looks good! Just a couple very minor wording suggestions.

File: modules/cert-manager-about.adoc (hunk 4 above)
Comment (subhtk) [reply modules/cert-manager-about.adoc:22]: Ack.

File: modules/cert-manager-about.adoc (hunk 3 above)
Comment (subhtk) [reply modules/cert-manager-about.adoc:20]: Ack.

**Comment by lunarwhite** [comment]
Sorry, I forgot to mention that this change is better applied to all active branches (4.12-4.16).

**Comment by JoeAldinger** [comment]
@ShaunaDiaz confirmed over slack that CPs to 4.12 are good. Updated labels and merging.

**Pull request merged**